### ISSUE TITLE
Run a re-import on staging

### ISSUE THREAD

**Issue by DiemBTran** [scientist-softserv/adventist-dl#92]
We want to reimport data on staging.  Before that is done, we want to:

- Track down why file sets are not attaching
- Ensure the metadata is mapping correctly
- When we have an existing work with an AARK and reimport, we don't create a new entry (e.g. we re-use the existing one)
- Remove the relationships that were created by each importer

**Comment by DiemBTran** [comment]
Linked items:

Blocks:

- Reimport data on dev

Is blocked by:

- Bulkrax FileSet attachment issue
- [2] Test each field using the QDC parser
- [2] Make Authors field ordered
- Run RemoveAccountRelationshipsJob on staging to remove relationships
- [3] Files associated with a work show up as a string (not human-readable) under Related URL
- Revisit Parser Mappings with 2022-10-27 Information and Later

**Comment by jillpe** [comment]
no longer relevant
